core/src/main/scala/algebra/PartialOrder.scala

```diff
@@ -158,6 +158,16 @@ object PartialOrder extends PartialOrderFunctions {
     }
 
   /**
+   * Define a `PartialOrder[A]` using the given functions `fs` until
+   * the first one returns a non-0 result or NaN.
+   */
+  def fromAll[@sp A](fs: ((A, A) => Double)*): PartialOrder[A] =
```

Comment:
fromFirst was the name @non used for the ticket, but I agree. In fact, I see the same problem with Order as well. Both Order and PartialOrder really determine the result via the first function that defines any order between the elements. Then again, I can equally accept reading it as "constructing an order derived from all of these functions used in turn". I'm impartial on fromAll or fromFirst, but I think it should be the same for Order and PartialOrder then.